stubs/AMQPQueue.php

```diff
@@ -350,4 +362,14 @@ public function getChannel()
     public function getConnection()
     {
     }
+
+    /**
+     * Get latest consumer tag. If no consumer available or the latest on was canceled empty string will be returned.
```

Comment:
I changed to `null` in `var_dump()` output and in `AMQPQueue::getConsumerTag()`.